code/game/machinery/dna_infuser/organ_sets/fly_organs.dm

```diff
@@ -0,0 +1,151 @@
+///bonus of the fly: you... are a flyperson now. sorry.
+/datum/status_effect/organ_set_bonus/fly
+	organs_needed = 4 //there are actually 7 fly organs that count, but you only need 4 to go full-flyperson. Be careful!
+	bonus_activate_text = null
+	bonus_deactivate_text = null
+
+/datum/status_effect/organ_set_bonus/fly/enable_bonus()
+	. = ..()
+	if(!ishuman(owner))
+		return
+	var/mob/living/carbon/human/new_fly = owner
+	if(isflyperson(new_fly))
+		return
+	//okay you NEED to be a fly
+	to_chat(new_fly, span_danger("Too much fly DNA! Your skin begins to discolor into a horrible black as you become more fly than person!"))
```

Comment:
`too much fly dna` is metagamey af, it should be some horrific to_chat about how your bones are contorting and shit

Reply:
true! really gotta get the fly movie vibes

Reply:
better: visible_message